components/AdvancedOptionsItem.qml

```diff
@@ -0,0 +1,184 @@
+                visible: button4.text
+            }
+        }
+        
+        ColumnLayout {
+            id: helpText
+            visible: false     
+            Layout.alignment: Qt.AlignTop | Qt.AlignLeft
+            
+            MoneroComponents.TextPlain {
+                id: helpTextLarge1
+                visible: true
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 13
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextLarge2
+                visible: helpTextLarge2.text
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 13
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextSmall1
+                visible: helpTextSmall1.text
+                Layout.leftMargin: 5
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 12
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextSmall2
+                visible: helpTextSmall2.text
+                Layout.leftMargin: 5
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 12
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextSmall3
+                visible: helpTextSmall3.text
+                Layout.leftMargin: 5
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 12
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextSmall4
+                visible: helpTextSmall4.text
+                Layout.leftMargin: 5
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 12
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
+                id: helpTextSmall5
+                visible: helpTextSmall5.text
+                Layout.leftMargin: 5
+                wrapMode: Text.WordWrap
+                font.family: MoneroComponents.Style.fontRegular.name
+                font.pixelSize: 12
+                color: MoneroComponents.Style.defaultFontColor
+            }
+            
+            MoneroComponents.TextPlain {
```

Comment:
Any reason for not doing `\n` instead of 6 textblocks?

Reply:
There are some messages in orange text that are displayed between the instructions lines if there is an error or if an information is needed.